README.md

```diff
@@ -1,167 +1,77 @@
-# Aerie CLI
-
-Note: this project is an informal CLI and is _not_ maintained by the MPSA Aerie team.
-
-##### Table of Contents
-
-- [Aerie CLI](#aerie-cli) - [Table of Contents](#table-of-contents)
-  - [Overview](#overview)
-  - [Getting Started](#getting-started)
-  - [Installation](#installation)
-    - [Installation with `pip`](#installation-with-pip)
-    - [Updating with `pip`](#updating-with-pip)
-    - [Installation with `poetry`](#installation-with-poetry)
-  - [Usage](#usage)
-    - [Setup](#setup)
-      - [With a Configuration File](#with-a-configuration-file)
-      - [Via the CLI](#via-the-cli)
-      - [Handling Authentication](#handling-authentication)
-    - [Commands](#commands)
-      - [Interactive](#interactive)
-      - [Non-Interactive](#non-interactive)
-      - [Help](#help)
-      - [Specific Sub-Commands: Plans](#specific-sub-commands-plans)
-        - [Download](#download)
-        - [Upload](#upload)
-        - [Duplication](#duplication)
-        - [Simulation](#simulation)
-      - [Specific Sub-Commands: Models](#specific-sub-commands-models)
-        - [Upload](#upload-1)
-        - [List](#list)
-        - [Delete](#delete)
-        - [Clean](#clean)
-    - [Python API](#python-api)
-      - [Basic Usage](#basic-usage)
-      - [Adding Methods](#adding-methods)
-      - [Advanced Authentication](#advanced-authentication)
-  - [Contributing](#contributing)
-    - [Contributor Installation](#contributor-installation)
-    - [Deployment](#deployment)
-    - [Dependency Management](#dependency-management)
-    - [Testing](#testing)
-      - [Unit Tests](#unit-tests)
-    - [Pre-Commit Hook](#pre-commit-hook)
-    - [IDE Settings](#ide-settings)
-      - [VS Code](#vs-code)
+# Aerie-CLI
 
----
-
-## Overview
+Aerie-CLI provides a command-line interface and user-extendable Python API for interacting with an instance of Aerie.
 
-Aerie CLI provides a command-line interface and user-extendable Python backend for interacting with an instance of Aerie.
-
----
+> Note: this project is an informal CLI and is _not_ maintained by the MPSA Aerie team.
 
 ## Getting Started
 
-This short procedure will get you up and running with the basics of Aerie CLI. Read the following sections for more information.
+This short procedure will get you up and running with the basics of the CLI. Read the following sections for more information.
 
-1. Install/update to Python >=3.9
+1. Install/update to Python >= 3.6
 
-2. Install Aerie CLI from Github:
+2. Install Aerie-CLI from Github:
 
    ```sh
-   python3 -m pip install git+https://github.com/NASA-AMMOS/aerie-cli.git@main
+   ➜  python3 -m pip install git+https://github.com/NASA-AMMOS/aerie-cli.git@main
```

Comment:
Wonderful feedback. I'm removing this from all the code blocks that can otherwise be copied. I left them in to larger code blocks that show example output to differentiate between the terminal and lines from the app.